imports/plugins/included/payments-stripe/server/methods/stripe.js

```diff
@@ -97,22 +97,43 @@ Meteor.methods({
     });
     check(paymentData, {
       total: String,
-      currency: String
+      currency: String,
+      shopId: String
     });
 
-    const chargeObj = {
-      amount: "",
-      currency: "",
-      card: {},
-      capture: true
-    };
+    // check if this is a seller shop for destination and transaction fee logic
+    const sellerShop = sellerShops.findOne(paymentData.shopId);
```

Comment:
Server side then yes, you'll need to access Shops directly as there is no SellerShops.
To get anything but the parent shop you can do something like
```js
Shops.findOne({
  $ne: {
     _id: Reaction.getShopId()
  }
});
```